library/Zend/Feed/Reader/Entry/Rss.php

```diff
@@ -201,7 +201,7 @@ public function getAuthors()
             );
         }
 
-        if (count($authors) == 0) {
+        if ($authors !== null && count($authors) == 0) {
```

Comment:
In the tests `$authors` would sometimes be `null` here, which `count` returns `0` on.